webserver/webserver/src/main/java/io/helidon/webserver/ServerRequest.java

```diff
@@ -142,6 +143,13 @@
      */
     Tracer tracer();
 
+    /**
+     * Closes the connection.
+     *
+     * @return Single completed when channel is closed.
+     */
+    Single<Void> close();
```

Comment:
I think this method name is not good. This would mean we are closing the request, where in fact it would close the underlying connection. I think `closeConnection` may be a better name, as it clearly states what is the intention of it.
This is the same for all the classes this was added to

Reply:
Also the documentation should clearly state what does this actually mean - is it immediately closing the connection? Is it adding `Connection: close` header? Will it close the connection after response is sent?
All mean "close connection", each in a very different way

Reply:
fixed